documentation/source/users/cantherm/input.rst

```diff
@@ -128,12 +127,11 @@ Parameter               Required?                   Description
 ``rotors``              optional                    A list of :class:`HinderedRotor()` and/or :class:`FreeRotor()` objects describing the hindered/free rotors
 ======================= =========================== ====================================
 
-The ``atom`` and ``bond`` parameters are used to apply atomization energy corrections (AEC), bond corrections (BC), and spin orbit corrections (SOC) for a given ``modelChemistry()`` (see `Model Chemistry`_).
+The types and number of atoms in the species are automatically inferred from the quantum chemistry output and are used
+to apply atomization energy corrections (AEC) and spin orbit corrections (SOC) for a given ``modelChemistry()``
+(see `Model Chemistry`_).
 
-Allowed atom symbols for the ``atoms`` parameter are
```

Comment:
We may still want to leave some description in the documentation of the atoms we commonly have AEC for